src/main/resources/plugin.yml

```diff
@@ -3,5 +3,5 @@ main: com.sk89q.craftbook.bukkit.CraftBookPlugin
 version: "${internalVersion}"
 dev-url: http://dev.bukkit.org/server-mods/craftbook/
 softdepend: [WorldEdit, WorldGuard, ProtocolLib, Vault]
-api-version: 1.13
+api-version: 1.17
```

Comment:
Please don't change the API version, we use WorldEdit's compat system

Reply:
Revert to old Settings